### ISSUE TITLE
aria-expanded should be set to "true" when dropdown is shown

### ISSUE THREAD

**Issue by kaelig** [jshjohnson/Choices#132]
Thank you for this great project, and glad to see accessibility has been considered!

I feel like the list of choices's container which has aria-expanded="false" should have it set to "true".

<img width="962" alt="screenshot 2017-03-27 18 09 44" src="https://cloud.githubusercontent.com/assets/85783/24384419/90f659a6-1318-11e7-8592-ef68dd27eab4.png">

**Comment by jshjohnson** [comment]
Ah ha! Good spot. I'll fix this ASAP.

**Comment by jshjohnson** [comment]
This has been resolved as of [version 2.7.7](https://github.com/jshjohnson/Choices/releases/tag/v2.7.7)!

**Comment by complanboy2** [comment]
@jshjohnson  is there a way that we can set this programmatically ? Not always needed the area-expanded as 'true' (it increases the page height)
